scripts/ingests/ingest_utils.py

```diff
@@ -322,7 +324,7 @@ def find_survey_name_in_simbad(sources, desig_prefix, source_id_index=None):
 
 
 # SPECTRAL TYPES
-def ingest_spectral_types(db, sources, spectral_types, references, regimes, spectral_type_error=None,
+def ingest_spectral_types(db, sources, spectral_types, references, regimes=None, spectral_type_error=None,
```

Comment:
I think regime is required to be not NULL so will having a default of None break things?